addons/project_hr_expense/models/project_project.py

```diff
@@ -76,20 +76,24 @@ def _get_expenses_profitability_items(self, with_action=True):
                 ('sheet_id.state', 'in', ['post', 'done']),
                 ('analytic_distribution', 'in', self.account_id.ids),
             ],
-            groupby=['currency_id'],
-            aggregates=['id:array_agg', 'untaxed_amount_currency:sum'],
+            groupby=['currency_id', 'untaxed_amount_currency'],
+            aggregates=['id:array_agg', 'analytic_distribution:array_agg'],
         )
         if not expenses_read_group:
             return {}
         expense_ids = []
         amount_billed = 0.0
-        for currency, ids, untaxed_amount_currency_sum in expenses_read_group:
+        for currency, untaxed_amount_currency_sum, ids, analytic_distribution in expenses_read_group:
+            analytic_contribution = analytic_distribution[0][str(self.account_id.id)] / 100
```

Comment:
An error arises when the same amount is used for multiple analytic distributions. For example, {'1,2,5,8': 100.0}